src/inetstack/protocols/layer4/tcp/established/sender.rs

```diff
@@ -458,12 +458,6 @@ impl Sender {
             );
         }
 
-        if cb.sender.unacked_queue.len() > 0 {
```

Comment:
I think @kyleholohan was printing this for some debugging but it might not be necessary any more.